scripts/comet_archiver

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3.7
+
+from comet import Archiver
+import argparse
+
+
+DEFAULT_DUMP_PATH = "./"
+DEFAULT_SCRAPE_INTERVAL = 360  # seconds
+
+
+def str2bool(v):
+    if v.lower() in ('yes', 'true', 't', 'y', '1'):
+        return True
+    elif v.lower() in ('no', 'false', 'f', 'n', '0'):
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
+
+def start(args):
+    comet = Archiver(args.dump_path, args.scrape_interval)
+    comet.run()
+
+
+parser = argparse.ArgumentParser(description="This is comet's archiver.")
```

Comment:
I kind of think we should standardise on `click` in future. We should probably write a chime dev guide with recommended tools (e.g. click for CLI, pytest for testing ...)

Reply:
:+1: 
https://github.com/chime-experiment/Pipeline/issues/28